Assets/Features/Workers/BuilderWorker.cs

```diff
@@ -48,7 +47,7 @@ public void MoveTo(Vector3Int target, Action onReachedPosition, Action onDone)
         /// </summary>
         private void CheckTaskExists(Vector3Int target, Action onDone)
         {
-            if (_grid.Get(target) == -1) onDone.Invoke();
+            if (GameManager.Instance.GridManager.Grid.Get(target) == -1) onDone.Invoke();
```

Comment:
-1 doesn't tell me anything about the state of the object. Could be beneficial to create a variable or something else that represents the -1 state, making it easier to read

Reply:
And move the functionality of an if statement to the next line